src/main.ts

```diff
@@ -16,9 +16,7 @@ async function run(): Promise<void> {
 
     const pluginVersionInput = core.getInput('plugin-version')
 
-    if (!checkForValidMillVersion(workingDirectory)) {
```

Comment:
This if was checking only if the promise was not defined since the result was not awaited. So the exception was never thrown.

Reply:
Ahhhh, yes indeed. Thanks!